conf/waives

```diff
@@ -0,0 +1,53 @@
+# TODO: unknown, but maybe expected as the rule description says that
+#   "The remediation for this rule uses firewall-cmd and nmcli tools.
+#    Therefore, it will only be executed if firewalld and NetworkManager
+#    services are running."
+# and since these are not running in an Anaconda %post environment,
+# maybe the remediation failing is expected .. ?
+# - see also https://github.com/ComplianceAsCode/content/issues/1639
+/hardening/anaconda/[^/]+/firewalld_sshd_port_enabled
+    rhel == 9 or rhel == 8
+
+# https://github.com/OpenSCAP/openscap/issues/1880
+# needs to be remediated more than once due to rule ordering issues
+/hardening/oscap/[^/]+/configure_bashrc_exec_tmux
+/hardening/oscap/[^/]+/no_tmux_in_shells
+/hardening/oscap/[^/]+/configure_usbguard_auditbackend
+    rhel == 9 or rhel == 8
+/hardening/oscap/[^/]+/configure_bashrc_tmux
+/hardening/oscap/stig(_gui)?/postfix_prevent_unrestricted_relay
+    rhel == 8
+
+# TODO: old test waives say this is also
+# https://github.com/OpenSCAP/openscap/issues/1880
+# but that seems to be incorrect as (unlike the others),
```

Comment:
The problem here is combination with `enable_authselect` rule. `accounts_password_pam_retry` passes during initial scan, `enable_authselect` forcibly sets authselect profile and that changes `accounts_password_pam_retry` result.

Reply:
Ok, I'll mention this in the comment. Thanks.